src/aind_behavior_video_transformation/etl.py

```diff
@@ -96,30 +103,89 @@ def _run_compression(
                     )
                     for params in convert_video_args
                 ]
+
+                failed_jobs = []
                 for job in as_completed(jobs):
                     result = job.result()
                     if isinstance(result, tuple):
-                        error_traces.append(result[1])
+                        # Job failed, store the original parameters for retry
+                        job_index = jobs.index(job)
+                        original_params = convert_video_args[job_index]
+                        failed_jobs.append(original_params)
+                        logging.warning(
+                            f"FFmpeg job failed for {original_params[0]}, "
+                            f"will retry with no compression. Error: {result[1]}"
+                        )
                     else:
                         logging.info(f"FFmpeg job completed: {result}")
+
+                # Retry failed jobs with no compression
```

Comment:
Sorry not super familiar with the whole workflow or how the videos are. But are you running with compression by default and then without compression if the job fails?